src/main/java/com/palantir/docker/compose/DockerComposeRule.java

```diff
@@ -187,6 +175,18 @@ public Builder saveLogsTo(String path) {
             return logCollector(FileLogCollector.fromPath(path));
         }
 
+        /**
+         * @deprecated Please use <code>shutdownStrategy(ShutdownStrategy.SKIP)</code> instead.
+         */
+        @Deprecated
+        public Builder skipShutdown(boolean skipShutdown) {
```

Comment:
Yeah, I wasn't sure whether to re-encode the default here as well as above... in the end I think this is good enough because the method is deprecated.  `skipShutdown(false)` is a weird thing to write anyway, so I'm kinda OK with slightly undefined behaviour here